.github/workflows/run_test_cases.yaml

```diff
@@ -118,6 +118,7 @@ jobs:
         # produces eunit.coverdata
       - name: eunit
         env:
+          IS_CI: "yes"
```

Comment:
we probably could set this on the job, or even workflow level

Reply:
Yes, setting it at the workflow level sounds like a better idea.